tck/features/clauses/call/Call1.feature

```diff
@@ -0,0 +1,280 @@
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Call1 - Basic procedure calling
+
+  Scenario: Standalone call to VOID procedure that takes no arguments
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+      """
+      CALL test.doNothing()
+      """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: Standalone call to VOID procedure that takes no arguments, called with implicit arguments
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+      """
+      CALL test.doNothing
+      """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: In-query call to VOID procedure that takes no arguments
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+      """
+      MATCH (n)
+      CALL test.doNothing()
+      RETURN n
+      """
+    Then the result should be, in any order:
+      | n |
+    And no side effects
+
+  Scenario: In-query call to VOID procedure does not consume rows
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    And having executed:
+      """
+      CREATE (:A {name: 'a'})
+      CREATE (:B {name: 'b'})
+      CREATE (:C {name: 'c'})
+      """
+    When executing query:
+      """
+      MATCH (n)
+      CALL test.doNothing()
+      RETURN n.name AS `name`
+      """
+    Then the result should be, in any order:
+      | name |
+      | 'a'  |
+      | 'b'  |
+      | 'c'  |
+    And no side effects
+
+  Scenario: Standalone call to procedure that takes no arguments and yields no results
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: ():
+      |
+    When executing query:
+      """
+      CALL test.doNothing()
+      """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: Standalone call to procedure that takes no arguments and yields no results, called with implicit arguments
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: ():
+      |
+    When executing query:
+      """
+      CALL test.doNothing
+      """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: In-query call to procedure that takes no arguments and yields no results
+    Given an empty graph
+    And there exists a procedure test.doNothing() :: ():
+      |
+    When executing query:
+      """
+      CALL test.doNothing() YIELD - RETURN 1
```

Comment:
why this "YIELD -" is not lligal using the antlr parser?

Reply:
I agree, I don't believe `-` should be a valid item after `YIELD`. I don't think that syntax was ever approved.
The specification ([CIP2015-06-24](https://github.com/openCypher/openCypher/blob/master/cip/1.accepted/CIP2015-06-24-call-procedures.adoc)) does not say anything about `YIELD -`, but the pull request that introduced that spec (#105) contains these test cases and a corresponding change to the grammar.
There seems to have been some inconsistencies, @boggle

Reply:
Thanks for pointing this out. We have to investigate this to see where this inconsistency originates from.

Reply:
After investigation, it was found that this is indeed a bug in the TCK and we should remove the scenario. I will make a suggestion to the PR in that regard.